### ISSUE TITLE
Not compatible with Xcode 6.4

### ISSUE THREAD

**Issue by gphilipp** [dmotz/natal#38]
I just installed Natal with required deps (node, leiningen …). I run `natal init AwesomeApp —interface om-next`. I got 

```
Updating Leiningen project
Creating React Native skeleton
Installing Pod dependencies
Updating Xcode project
Creating Natal config
Cannot find device `iPhone 6s`
```

I can’t launch any Natal command inside the directory, it complains about the missing config file: `No Natal config was found in this directory (.natal)

```
natal —version
0.1.12
```

I'm using Xcode 6.4. 

[Others](https://clojurians.slack.com/archives/cljsrn/p1448295230000526) seem to have this issue.
In the simulator, there's no reference to `iPhone 6s`, only up to `iPhone 6 plus`. Upgrading to Xcode 7 resolved the issue, but @mfikes [says](https://clojurians.slack.com/archives/cljsrn/p1448401353000574)  it should work with 6.3+.

**Comment by mfikes** [comment]
The 6.3+ requirement arose from the use of newer Objective C source-level constructs in React Native.

Perhaps Natal is now explicitly looking for a device by name that doesn't exist in Xcode 6.3?

**Comment by dmotz** [comment]
Just published `0.1.13` which defaults the simulator to iPhone 6, which should fix this issue. Let me know if it doesn't and we'll reopen.
